common/src/main/java/org/dash/wallet/common/ui/BaseAdapter.kt

```diff
@@ -1,36 +0,0 @@
-package org.dash.wallet.common.ui
-
-import android.annotation.SuppressLint
-import android.view.View
-import android.view.ViewGroup
-import androidx.annotation.LayoutRes
-import androidx.recyclerview.widget.DiffUtil
-import androidx.recyclerview.widget.ListAdapter
-import androidx.recyclerview.widget.RecyclerView
-
-abstract class BaseAdapter<T : Any,>(
-    diffCallback: DiffUtil.ItemCallback<T> = object : DiffUtil.ItemCallback<T>() {
-        override fun areItemsTheSame(oldItem: T, newItem: T) =
-            oldItem == newItem
-
-        @SuppressLint("DiffUtilEquals")
-        override fun areContentsTheSame(oldItem: T, newItem: T) =
-            oldItem == newItem
```

Comment:
Generally, `areItemsTheSame` and `areContentsTheSame` should not have the same comparing logic - it makes the whole DiffUtil and the adapter useless since any change to the item contents causes the whole list to redraw.

`areItemsTheSame` should compare items identifiers (unique type, transaction hashes, etc) to check if it's the same item (Coinbase vs Uphold), while `areContentsTheSame` should compare fields of the items to know when to redraw a specific row.